examples/structured_2d_dgsem/elixir_eulerpolytropic_isothermal_wave.jl

```diff
@@ -0,0 +1,83 @@
+
+using OrdinaryDiffEq
+using Trixi
+
+###############################################################################
+# semidiscretization of the polytropic Euler equations
+
+gamma = 1.0
```

Comment:
Maybe note in the comment here that this is an isothermal run because you set `gamma = 1.0`?

Reply:
Done.